client/src/featureform/get_local.py

```diff
@@ -0,0 +1,400 @@
+from .format import *
+from .sqlite_metadata import *
+
+def get_user_info_local(name):
+    db = SQLiteMetadata()
+    users_list = db.getVariantResource("users", "name", name)
+    if users_list == []:
+        print("User not found.")
+        return
+    user = users_list[0]
+    format_rows("USER NAME: ", user["name"])
+    format_rows("TYPE: ", user["type"])
+    format_rows("STATUS: ", user["status"])
+    return user
+
+def get_entity_info_local(name):
+    db = SQLiteMetadata()
+    entity_list = db.getVariantResource("entities", "name", name)
+    if entity_list == []:
+        print("User not found.")
+        return
+    entity = entity_list[0]
+    
+    returned_features_list = get_related_resources("feature_variant", 
```

Comment:
What does get_related_resources() do?

Reply:
Does it only work for features? Would get_feature_resources be better?

Reply:
it gets all the related resources of a specific resource (i.e. all labels related to a source, or all features related to a training set(